main.js

```diff
@@ -1,28 +1,79 @@
-function generateBoard() {
-  var $board = document.createElement('div')
+function Tile(row, column, isHidden) {
+  this.row = row
+  this.column = column
+  if (row === 0 || row === 7 || column === 0 || column === 7) {
+    isHidden = true
+  }
+  else {
+    isHidden = false
+  }
+}
+
+function generateTiles() {
```

Comment:
Same here, unfortunately, ESLint did not catch the part of your code that mutates the `window` object.